includes/wikia/GlobalTitle.php

```diff
@@ -835,6 +844,32 @@ private function loadNamespaceNames() {
 	}
 
 	/**
+	 * Get $wgExtraNamespaces for title language
+	 *
+	 * @return array
+	 */
+	private function getLocalWgExtraNamespaces() {
+		global $wgExtensionNamespacesFiles;
+		$extraNamespaces = [ ];
+
+		foreach ( $wgExtensionNamespacesFiles as $extFile ) {
```

Comment:
Please cache reading from all these files in a `static` variable so that multiple calls will not do I/O over and over.